packages/webapp/src/views/common/legal-agreement-summary.html

```diff
@@ -46,6 +47,16 @@
         'classes': 'govuk-!-margin-bottom-7',
         'show':isLegalAgreementTypeS106Type
     },
+    {
+        'text': 'Other landowners or leaseholders on conservation covenant',

```

Comment:
This does not cater for planning obligations.
![image](https://github.com/DEFRA/biodiversity-net-gain-service/assets/8915039/b7fa51e4-5ac9-4774-a571-2d8cb5dd4141)